index.js

```diff
@@ -1,25 +1,175 @@
+    <img
+      class="works-card--img"
+      src=${item.image}
+      alt="project pic"
+    />
+    <div class="works-card--desc">
+      <h2 class="works-card--title">${item.name}</h2>
+      <ul class="works-card--info">
+        <li>${item.info.company_name}</li>
+        <li>
+          <img src="images/works_images/Counter.png" alt="dot icon" />
+        </li>
+        <li>${item.info.position}</li>
+        <li>
+          <img src="images/works_images/Counter.png" alt="dot icon" />
+        </li>
+        <li>${item.info.date}</li>
+      </ul>
+      <p class="works-card--detail">
+        ${item.description}
+      </p>
+      <ul class="works-card--languages">
+         ${technologies}
+      </ul>
+      <button class="portfolio-button" type="button" onclick="openDetailWindow(${item.id})">See Project</button>
+    </div>
+    `;
+    document.getElementById('portfolio').appendChild(card);
+  });
+}
+function openDetailWindow(id) {
+  const openedDetail = projects.filter((item) => item.id === id.toString())[0];
+  const technologies = openedDetail.technologies
+    .map((technology) => `<li>${technology}</li>`)
+    .join('');
+  const detailContainer = document.querySelector('.detail-container');
+  detailContainer.innerHTML = `
+  <div class="detail-card">
+  <div class="upper-part">
+  <div class="first">
+    <h2 class="works-card--title">${openedDetail.name}</h2>
+    <button type="button" class="close-detail" onclick="closeDetail()">
+      <img src="images/closeDetail.png" alt="close icon" />
+    </button>
+  </div>
+  <ul class="works-card--info">
+  <li>${openedDetail.info.company_name}</li>
+  <li>
+    <img src="images/works_images/Counter.png" alt="dot icon" />
+  </li>
+  <li>${openedDetail.info.position}</li>
+  <li>
+    <img src="images/works_images/Counter.png" alt="dot icon" />
+  </li>
+  <li>${openedDetail.info.date}</li>
+  </ul>
+</div>
+<img src="images/works_images/detail.png" alt="project pic" />
+<div class="bottom-part">
+  <p class="works-card--detail">
+  Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the release.
+ </p>
```

Comment:
[OPTIONAL] Well done on the popup 🎉 but in order to make it look better and cooler 😎  the Modal popup data should be populated with the data from the projects array. 📑 

Although I understand the reason why you did that, probably to make the cards the same size as requested by the previous reviewer. 🤓

Reply:
@didierganthier Thanks for the review, and I will do that.